tests/loggers/test_csv.py

```diff
@@ -95,7 +94,6 @@ def test_file_logger_log_hyperparams(tmpdir):
         "bool": True,
         "dict": {"a": {"b": "c"}},
         "list": [1, 2, 3],
-        "namespace": Namespace(foo=Namespace(bar="buzz")),
```

Comment:
This requires `unsafe_load`

Reply:
so we always need to convert namespaces to dicts now?

Reply:
Yes: https://github.com/yaml/pyyaml/issues/482